src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= LONG_TERM) {
+    totalCost -= LONG_TERM_DISCOUNT;
+  } else if (days >= SHORT_TERM) {
+    totalCost -= SHORT_TERM_DISCOUNT;
+  }
```

Comment:
You can remove the `else` keyword to follow the checklist item 5. Since you're returning immediately in the `if` block, there's no need for an `else` block; you can simply use another `if` statement. This also adheres to the checklist item 4 which prefers `if` with `return` over `if else` to simplify conditions.